src/ios/Calendar.m

```diff
@@ -392,8 +392,9 @@ - (NSMutableArray*) eventsToDataArray: (NSArray*)matchingEvents {
     if (event.location != nil) {
       [entry setObject:event.location forKey:@"location"];
     }
+    NSString *notesWithURL = [NSString stringWithFormat:@"%@ %@", event.notes, event.URL];
```

Comment:
The code following this one checks if notes are nil. This code should also be protected, in order to not print nil (both for notes and URL).

Reply:
Yes, good catch :)